### DISCUSSION TITLE
How to clear OT trash bytes using batch editor

### DISCUSSION THREAD

**Discussion by Joseph11024** [kwsch/PKHeX#4355] [Q&A]
Hello, I’m trying to figure out how to clear the OT trash bytes using the batch editor. I found this forum talking about it but I don’t quite understand what it’s saying I should do so just wondering if anyone could help me out. https://projectpokemon.org/home/forums/topic/55988-batch-editor-nickname_trashot_trash/

**Comment by kwsch** [comment]
Mirror what the values would look like in the trash bytes editor:
```
=OriginalTrainerName=Nik
.OriginalTrainerTrash=$[]4E,00,69,00,6B,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00,00
```

**Reply by Joseph11024** [reply]
Ok perfect thank you it worked. Thank you for explaining.
